.github/workflows/release.yml

```diff
@@ -0,0 +1,32 @@
+name: Release
+on:
+  push:
+    branches:
+      - main
+
+permissions:
+  contents: read # for checkout
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write # to be able to publish a GitHub release
+      issues: write # to be able to comment on released issues
+      pull-requests: write # to be able to comment on released pull requests
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: "18"
+      - name: Install Plugins
+        run: npm i -g semantic-release @semantic-release/changelog @semantic-release/commit-analyzer conventional-changelog-conventionalcommits
```

Comment:
Shouldn't this be bun instead of node or is there no bun alternative to this as of now?

Reply:
I tried bun, some packages like @semantic-release/changelog throws error upon installing. There documentation also not have bun examples.

```
@sub1120 ➜ /workspaces/finance-management-api (setup/semantic-release) $ bun i @semantic-release/changelog
[0.06ms] ".env"
bun add v1.1.38 (bf2f153f)
error: Integrity check failed for tarball: cssesc

error: IntegrityCheckFailed extracting tarball from cssesc
```